app/javascript/components/common/Introducer.tsx

```diff
@@ -30,17 +31,13 @@ export const Introducer = ({
     },
     {
       onSuccess: () => {
-        setHidden(true)
+        ref.current!.parentElement!.classList.add('hidden')
```

Comment:
DJ suggested doing this in `useLayoutEffect`, and keeping the hidden state, but as this is directly powered by this callback and the state cannot be changed back, the extra indirection felt unnecessary to me. However, there may be good reasons to do that instead of what I've suggested so @kntsoriano please feel to refactor and @SleeplessByte please feel free to shout at me for not doing exactly what you said :)

Reply:
In this case the `layoutEffect` was only necessary to sync the state of the "hidden" variable. IN this case we don't need that because the effect is imperative. What you did here is fine.

Reply:
Ok great. Thank you 🙂